index.d.ts

```diff
@@ -115,6 +115,7 @@ interface AccordionProps extends ViewProps {
     color?: string;
   };
   position?: "top" | "bottom";
+  hideCustomToggleAction?: boolean;
```

Comment:
@Tsudhishnair Naming convention is confusing. Let me know if I misunderstood.
So when this flag is true, the default toggle will be hidden, right? If yes then I can use 
`shouldShowToggle` as true by default in the library? and dev can pass `shouldShowToggle` as false in case of custom toggle.